cv_next/app/feed/components/CVItem.tsx

```diff
@@ -27,62 +22,107 @@ export default function CVItem({ cv }: CVCardProps) {
 
   const getBlur = useMemo(
     () => async (imageURL: string) => {
-      return await getBlurredCv(imageURL);
+      const response = await fetch("/actions/cvs/fetchPreviews", {
```

Comment:
Relevant to all the fetching functions here: a) please replace the hardcode uris with constants; preferably you can create a route uri generator in the utils. b) please switch to hooks, or some form of encapsulation for these methods as they unnecessarily clutter up this component.